README.md

```diff
@@ -16,7 +16,6 @@ S2 documentation can be found on [s2geometry.io](http://s2geometry.io).
 
 ## Requirements for End Users
 
-* A POSIX system (for getrusage).
```

Comment:
Maybe say "A POSIX system (for getrusage) or Windows."

Reply:
Fixed.